Authentication/readme.md

```diff
@@ -0,0 +1,110 @@
+# Juntos Auth Service
```

Comment:
Add some diagrams showing the usage of the module in various services (reference the System design doc to see related services).

Reply:
Is this addressed in the new changes?